### PR TITLE
Cmake sync master misc

### PR DESCRIPTION
**Pull request by SebastianBoe**

### CODE REVIEW

**Comment by SebastianBoe** [comment]
oops, forgot 1 patch

File: drivers/CMakeLists.txt [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-add_subdirectory_ifdef(CONFIG_NET_L2_ETHERNET   ethernet)
 add_subdirectory_if_kconfig(flash)
 add_subdirectory_if_kconfig(gpio)
 add_subdirectory_if_kconfig(grove)
 add_subdirectory_if_kconfig(i2c)
 add_subdirectory_if_kconfig(ieee802154)
-add_subdirectory(interrupt_controller)
 add_subdirectory_if_kconfig(ipm)
+add_subdirectory_if_kconfig(led_strip)
+add_subdirectory_if_kconfig(net)
```
Comment (nashif) [inline drivers/CMakeLists.txt]: I do not think we have CONFIG_NET, so this should be plain add_subdirectory(net)

**Review by nashif — DISMISSED** [review]

File: drivers/CMakeLists.txt (hunk 1 above)
Comment (SebastianBoe) [reply drivers/CMakeLists.txt]: Good catch. This was a mistake.

Comment (SebastianBoe) [reply drivers/CMakeLists.txt]: I ifdef'd on CONFIG_NETWORKING to align with KBuild.

**Review by nashif — APPROVED** [review]

**Pull request merged**